torch/csrc/jit/passes/create_functional_graphs.h

```diff
@@ -12,9 +12,12 @@ TORCH_API void CreateFunctionalGraphs(const std::shared_ptr<Graph>& graph);
 
 TORCH_API void InlineFunctionalGraphs(const std::shared_ptr<Graph>& graph);
 
-// Replaces in-place aten ops with their functional equivalents when it can
-// be proven that this does not change graph semantics.
-TORCH_API void RemoveMutation(const std::shared_ptr<Graph>& graph);
+// Replaces list mutation & in-place aten ops with their functional equivalents
+// when it can be proven that this does not change graph semantics.
+TORCH_API void RemoveMutation(
```

Comment:
Can we have separate passes for removing list and tensor mutation? They seem like pretty different optimizations to me, although they share a common implementation. This API seems prone to mis-use (mixing up the arguments, forgetting to specify a defaulted argument, etc.)